projects/cashmere/src/lib/toaster/hc-toast.component.scss

```diff
@@ -128,7 +130,7 @@ $hc-header-font-size: 18px !default;
HWOuICqO9GYO9DWOtIyuM9iYM9vqMtWZQGO3VGex1GWvNpDDaqzLY6zHWmlFhtFdjsNdhrDWzwmivwmAvz1hLRnsVBntZxlr6pjDaizLYyzHW0nOF0V6MwV6GsZbOK4z2Igz2/Yy19LbCaN/NYN/HWGfqLx/KUhjtu/y49wIGZqwzNfDh0b/XPsvQGfXlnx9e+ya9zCPs2xjrTM33IfiAR9qJCo+0b2Kwr2esMzUvH7UZ7UyF0b6awb6Osc7UvP5XbKOdqTDaVzHYlzPWmZrLng812pkKo30xg30ZY52pue7FK6OdqTDaFzHYbzPWmZrb3mlgtDMVRvtNBvt1xjpTc9/bwox2psJov8pgn2esMzXLvIfXaGcqjPZZBvtlxjpTs+wJF0Y7U2G0X2SwnzPWmZp1zo4z2pkKo/2Mwf6esc7UrHsqs9HOVBjt7xjsb4x1pmab604Y7UyF0f7KYD8w1pmabS8SZLQzFUYbMNhgrFM1+1zRzWhnKoz29ME21pmafS+/abQzFZNHe+ZgG+tMTca1ko12pmLiaM8abGOdqcmI9YnRzlRMGu0Zg22sMzVZsT4x2pmKCaM9W7CNdaYmM9YnRjtTMVm0Zwq2sc7UZMf6xGhnKiaK9izBNtaZmjFifWK0MxWTRHuGYBvrTM1YsT4x2pmKCaJ99GAb60zNmLE+MdqZioNH+8jBNtaZmrFjfWK0MxUHjvZRg22sMzXHiPWJ0c5UHDTaRwy2sc7UHCvWJ0Y7U3HAaB8t2MY6U3PMWJ8Y7UzFwaJ9pGAb60zNsWN9YrQzFQeK9lGCbawzNXPE+sRoZyoOEu0jBNtYZ2rmivWJ0c5UHCDaowfbWGdq5oz1idHOVAwe7ZGDbawzNXPH+sRoZyoGjvaowTbWmRpj/ZjRzlQMGu0Rg22sMzXG+iVGO1MxYLRHC7axztQY69cY7UzFYNEeKdjGOlNjrC9htDMVA0V7lGAb60yNsb6G0c5UDBLtEYJtrDM1xvoWRjtTMUC004NtrDM1xvoeRjtTER7t5GAb60yNsV6C0c5UBEc7NdjGOlNjrJdktDMVodFODLaxztQY6zUY7UxFYLTTgm2sMzXGek1GO1MRFu2kYBvrTI2x3oLRzlQERTsl2MY6U2Ost2S0MxUh0U4ItrHO1BjrPRjtTEVAtPcOtrHO1BjrPRntTMXO0d4z2MY6U2OsExjtTMWO0d4r2MY6U2OskxjtTMVO0d4j2MY6U2OsExntTMUO0d462MY6U2OskxntTMXG0d4y2MY6U2OsR2C0MxUbRnurYBvrTI2xHonRzlRsFO0tgm2sMzXGekRGO1OxQbTXDraxztQY65EZ7UzFytFeM9jGOlNjrI/AaGcqVoz2WsE21pkaY30kRjtTsVK01wi2sc7UGOsjMtqZihWivXSwjXWmxlgfmdHOVCwc7SWDbawzNcZ6BkY7U7FgtJcKtrHO1BjrmRjtTMVC0V4i2MY6U2OsZ2S0MxULRPveYBvrTI2xnpnRzlTcGe17gm2sMzXGWkY7VXFHtG8NtrHO1BhrfWO0MxU3RvuWYBvrTI2x1nNGO1NxQ7SvDbaxztQYa51ntDMVV0b7mmAb60yNsdbbjHam4opoXxpsY52pMda6nNHOVFwY7UuCbawzNcZa1zPamYoLov1WsI11psZY63ZGO1PxRrRfC7axztQYa93PaGcqXon2uWAb60yNsdZyjHam4ky0Xwr2zxjrRI2x1vKMdqbiocXfeRrs98DvW6xGV2mMtdZjtDP9Nw9N/uppsP8J/LTZcnSJxlhrfUY7z3vgvx5/4ocnX/zHpsvRWxpjre0Y7Ty/8ego+3Gwf+XJ4bd21Rhrbc9oZ3nPoxcgnwZbGRpjrf0Y7Sy/nB48DvazVyS1i8ZYa39GO8eLwfbFxv01xlo5jHaGF5/D1r4aY608RjuIwc7QGGvlMtohHgf7791WMbfGWCuf0d7Pp9ODx8H+a4eFzK4x1hqH0d7Hx9ODH176pDbRGGuNx2hv78VgG4/tNM5b4zLa2/raisfB/gT8uf1aptMYa43PaG/jA2eewwb4z+MvanGNsdZxGO11fQL+/fgTT4P9Cfhjs+XMpTHWOh6jvZ4/eHIA/dL7sD/iBiytMdY6LqO9vOaFN4KcO3HGDVhOY6x1fDZjOc2ZZrx2pqMbcL/GWGseNuN+zSvNeOvUdDfgdo2x1nxsxu2aN5pxybVE3IDrNcZa87IZ12suaMalF39yAy7XGGvJZlyuubAZ11ytzw14W2OspROb8bbmimZce3lVN+C8xlhLT9mM85orm3HL9bDdgOcaYy2dYzOea25oxq03MHADvmmMtfQWm/FNc2Mz7rnjjBtgrKVr2Iw7m3HvLcJm3oDGWEvXshl3WOKejjNuQGOspVvZjBstdRPemTagMdbSvWzGDZa8a/oMG9AYa2kpNuNKSwYbjr0BjbGWlmYzrrB0sOGYG9AYa2ktNuNCawQbjrUBjbGW1mYzLrBWsOEYG9AYa2krNuMNawYbxt6AxlhLW7MZr1g72DDmBjTGWtqLzThji2DDWBvQGGtpbzbjBVsFG8bYgMZYSylsxhNbBhuyN6Ax1lIam/HI1sGGzA1ojLWUymZ8sUewIWsDGmMtpbMZ7BdsyNiAxlhLo5i+GXsGG/bdgMZYS6OZuhl7Bxv22YDGWEujmrYZCcGGbTegCRi8pLtM2YyUYMM2G9CEDF7S3aZrRlKwYd0NaIIGL2kRUzUjLdiwzgY0YYOXtJhpmvHj3gs44zSoWuBnNYGDl7So0//j/wLe3/mzmtBmvPv8+fPDg3fvdl7Ki34Bfue2DfgE/AF8XHRFkpL9zEMzfrrhz8Y242unw4MND7H+L+AfV/yZP4H/8LABkubyHvgnD8249GDvA/BvQpsxUrBP3gO/8XDU/Qvfb8QnHn4rfuRh8JFDl7Sp98CvXz5+5vuj7r+BvxikGc+CLUnKlvguEUnSC/4fXfUCq3Li8McAAAAASUVORK5CYII=');
```

Comment:
My IDE changed this from double quotes to single quotes, sorry for the massive diff.